src/jsxstyle-components.js

```diff
@@ -0,0 +1,9 @@
+// Mostly copied from:
+// https://github.com/threepointone/glamor/blob/
+// 5e7d988211330b8e2fca5bb8da78e35051444efd/src/jsxstyle.js
+/*
+ * This is used to generate the helper functions
+ * like: glamorous.block and the <glamorous.Block /> elements
+ */
+
+export default ['view', 'block', 'inlineBlock', 'flex', 'row', 'column']
```

Comment:
Ah, that gives me an idea, what if we did:

```javascript
import { Block, Flex, Row, Column } from 'glamorous/display'
```

Might be a little bit of a challenge for the UMD build though 🤔  We'd either have to have an entirely different file you'd have to load, or still tack it onto `glamorous.display`...

Hmmm... I think that I just like this the best:

```javascript
const {A, Img, Div} = glamorous
const {Block, Flex, Row, Column} = glamorous.display
const MyDiv = glamorous.div({/* styles */})
```

It suffers from 0 UMD issues, and it makes a clear separation between the original use-case of elements and the "display" use case of display elements.